service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/FurtherEvidenceUploadedEmailContentProvider.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.furtherevidence.FurtherEvidenceDocumentUploadedData;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.AbstractEmailContentProvider;
+
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
+
+@Service
+public class FurtherEvidenceUploadedEmailContentProvider extends AbstractEmailContentProvider {
+    public FurtherEvidenceDocumentUploadedData buildParameters(CaseData caseData, String sender) {
+        return FurtherEvidenceDocumentUploadedData.builder()
+            .caseUrl(getCaseUrl(caseData.getId()))
+            .callout(buildCallout(caseData))
```

Comment:
build callout is not correct here
If you see implementation of buildCallout it internally prefix text with "^"
but email template already define callout, so thats why you have "^^"